### ISSUE TITLE
Feature proposal: highlight last update of display alert if it's unexpectedly late

### ISSUE THREAD

**Issue by bonjour81** [poblabs/weewx-belchertown#127]
Hello !
A few years ago, I built my own weewx skin. 
At that time, I added a feature that might be nice to add in Belchertown skin too.

It was simply to display  a "red" message in page header if last update was some time ago (figuring something is not working  and updates are stopped for  x days, y hours etc....).
This way, you can figure out at a glance that something is wrong (ftp not working, weewx dead etc...)

In my older skin, I was doing this by a few lines of php, but I assume a simple js could do it.

Basically the algorithm could looks like:
if $current.dateTime.raw < ( now() - 30minutes) then 
 echo in red/bold  "NO UPDATE SINCE: now() - $current.dateTime.raw "  (with a nice format).
else 
echo in default style "Last updated on $current.dateTime.raw"   # same as currently available

**Comment by poblabs** [comment]
I assume this would only apply to those who are not using mqtt websocket?

**Comment by HoracioDos** [comment]
Hello.
It would be great if you can control it with a keyword like "show_noupdates  = 1" or something like that, only for non websockets config.

**Comment by bonjour81** [comment]
Hi!
No, I wasn't meaning for mqtt but for standard generation/upload (maybe mqtt too).

Say your ftp or weewx or whatever does not upload anymore files to your website since yesterday.
Of course it will be written on your page "last update 15 april 15h20" 

But to be honest, it's not really eye catching, and you may not notice it's stopped since yesterday.
I think it's sometime usefully if, after several hours without updates, it's display with some emphasis (red color, or whatever).

I see this as a diagnosis feature, just like not filling gaps on all curves (so you can see it a sensor is interrupted)

**Comment by poblabs** [comment]
@bonjour this is implemented. please try it out and let me know of any bugs by re-opening this issue. It's working for me pretty well, so I'm closing this for now 😃 

skin.conf option:
`show_last_updated_alert = 0` - set to 1 to enable

`last_updated_alert_threshold = 1800` this is seconds - so 1800 is 30 minutes

label `header_last_updated_alert` is the alert text that will show.
